libs/article/api/src/lib/article.service.ts

```diff
@@ -285,12 +285,15 @@ export class ArticleService {
       }
     })
 
+    const articlePublishedAt =
```

Comment:
Fixes https://wepublish.atlassian.net/browse/TSRI-134

When the article.publishedAt doesn't exist or is older than the new publishing date, take the new publishing date.